docs/developer-docs/latest/development/custom-fields.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Custom fields reference - Strapi Developer Docs
+description: Learn how you can use custom fields to extend Strapi's content-types capabilities.
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/custom-fields.html
+---
+
+# Custom fields
+
+<!-- TODO: add links to user guide section(s) somewhere -->
+Custom fields extend Strapi’s capabilities by adding new types of fields to content-types. Once created or installed, custom fields can be used in the Content-Types Builder and Content Manager just like built-in fields.
```

Comment:
```suggestion
Custom fields extend Strapi’s capabilities by adding new types of fields to content-types or components. Once created or installed, custom fields can be created in the Content-Type Builder and used in the Content Manager just like built-in fields.
```

Reply:
> Once created or installed,

I think you could just say "Once installed" but what you have is fine too